src/main/java/EPstorage/PlaylistCommands.java

```diff
@@ -0,0 +1,152 @@
+package EPstorage;
+
+import object.MovieInfoObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.TreeMap;
+
+/**
+ * class that contains all methods that deal with individual Playlist object and the list of Playlist objects
+ */
+public class PlaylistCommands {
+    private ArrayList<Playlist> playlists;
+    private EditPlaylistJson editPlaylistJson;
+
+    public PlaylistCommands(ArrayList<Playlist> playlists) throws IOException {
+        editPlaylistJson = new EditPlaylistJson();
+        this.playlists = playlists;
+    }
+
+    /**
+     * add new Playlist object into the list
```

Comment:
Javadoc comments should start with a **singular verb** `Adds` (mind the caps) and ends with a **full stop**.